### PR TITLE
Upgrade Teleport dependency to v6.

### PR DESCRIPTION
**Pull request by marshall-lee**
Move to current `branch/v6` (gravitational/teleport@693576f98979)

This is an extract from #196

### CODE REVIEW

**Comment by klizhentas** [comment]
@marshall-lee please rebase and merge

**Comment by marshall-lee** [comment]
@klizhentas review approval required

**Comment by Joerger** [comment]
@marshall-lee It looks like `github.com/gravitational/teleport` is still being imported. Most if not all of the values needed should be in the api-package. If you'd like I can make a PR off this this branch To fix that since I know where each value should be located.

File: access/mattermost/bot.go [hunk 1]
```diff
@@ -2,204 +2,304 @@ package main
...
-	actionsAttachment, err := b.NewActionsAttachment(reqID, reqData, "EXPIRED")
-	if err != nil {
-		return trace.Wrap(err)
-	}
+		result := resp.Error()
+		if result == nil {
+			return nil
+		}
+
+		logrus.Debugf("RESP BODYYYY %q", string(resp.Body()))
```
Comment (klizhentas) [inline access/mattermost/bot.go:110]: @marshall-lee remove the line please

**Review by klizhentas — APPROVED** [review]

**Pull request merged**